core/types/block.go

```diff
@@ -93,7 +93,9 @@ type Header struct {
 	AuRaStep uint64
 	AuRaSeal []byte
 
-	BaseFee *big.Int `json:"baseFeePerGas"` // EIP-1559
+	BaseFee         *big.Int     `json:"baseFeePerGas"`   // EIP-1559
```

Comment:
Does this mean Erigon haven't supported EIP-1559 yet?

Reply:
no it was there before this field just got reformatted due to comment indendation which is why it's showing up as a diff.